src/app/frontend/chrome/errordialog_service.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Returns the error dialog service
+ *
+ * @param {!md.$dialog} $mdDialog
+ * @return {!Object}
+ */
+export default function create($mdDialog) {
+  return new ErrorDialog($mdDialog);
+}
+
+/**
+* Error dialog to display a errors that occured in the backend
+* @final
+**/
+export class ErrorDialog {
```

Comment:
I think you should just create a class without the factory function. The reason is that the factory function does nothing, besides being an additional layer. Then use `.service` instead of `.factory` in the module. What do you think?